packages/react-component-library/e2e/DatePicker/default.spec.ts

```diff
@@ -44,3 +44,46 @@ test.describe('DatePicker, default', () => {
     })
   })
 })
+
+test.describe('DatePicker, jump to today', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(
+      '/iframe.html?id=components-date-picker--jump-to-today&viewMode=story'
+    )
+  })
+
+  test('jumps to today when button is clicked', async ({ page }) => {
+    await page.click(selectors.button)
+
+    const prevMonthButton = page.getByRole('button', {
+      name: /go to previous month/i,
+    })
+    await prevMonthButton.click()
+    await prevMonthButton.click()
+    await prevMonthButton.click()
+
+    const initialMonth = await page
+      .locator(selectors.monthHeading)
```

Comment:
BTW (if not aware) Playwright [no longer encourages use of the locator method](https://playwright.dev/docs/locators) (originally it was all they had...)